src/ralph/export_to_ng/subscribers.py

```diff
@@ -61,6 +70,18 @@ def _get_publisher_signal_info(func):
     }
 
 
+def _handle_custom_fields(data, obj):
+    for symbol, value in data['custom_fields'].items():
```

Comment:
maybe `data.get('custom_fields', {}).items()`? Otherwise we have to release ng first - make it backward compatible.